app/views/shared/signup_confirmation.html.erb

```diff
@@ -10,21 +10,20 @@
   </script>
 <% end %>
 
-<div class="CDB-Text <%= @organization.present? ? 'Sessions' : 'Sessions-navy' %>" style="<%= @organization.present? ? background : '' %>">
+<div class="Sessions">
```

Comment:
It appears you are reverting the file to match blp_dev on bloomberg/cartodb.  Is there a reason the background style was left out of the Sessions div class?  https://github.com/bloomberg/cartodb/blob/blp_dev/app/views/shared/signup_confirmation.html.erb

Reply:
Reviewed offline with @mbektas.  Adding a background causes problems with new merge code.  However it works fine with it removed.